app/views/casa_cases/_volunteer_assignment.html.erb

```diff
@@ -41,20 +41,20 @@
               </td>
               <td data-test="action">
                 <% if policy(assignment).unassign? %>
-                  <%= button_to(t(".button.unassign"),
+                  <%= button_to("Unassign Volunteer",
                                 unassign_case_assignment_path(assignment),
                                 method: :patch,
                                 class: "btn btn-outline-danger text-wrap") %>
                 <% elsif !assignment.volunteer.active? && policy(assignment.volunteer).activate? %>
-                  <%= link_to t(".button.activate"),
+                  <%= link_to "Activate Volunteer",
                               activate_volunteer_path(assignment.volunteer,
                                                       redirect_to_path: 'casa_case',
                                                       casa_case_id: @casa_case.id
                               ),
                               method: :patch,
                               class: "btn btn-outline-success" %>
                 <% else %>
-                  <%= t("common.none") %>
+                  "None"
```

Comment:
I think this one was not supposed to have quotes around it.  However, i am having issues making an update. I can try to address this later.

Reply:
OK, I got this updated.